### ISSUE TITLE
Resolve try-except-pass warnings

### ISSUE THREAD

**Issue by afuetterer** [ddionrails/ddionrails#120]
https://docs.openstack.org/bandit/latest/plugins/b110_try_except_pass.html

see: 
https://github.com/ddionrails/ddionrails/search?q=except:

**Comment by hansendx** [comment]
[List of except blocks on codacy](https://app.codacy.com/app/ddionrails/ddionrails/issues?bid=10077049&filters=W3siaWQiOiJMYW5ndWFnZSIsInZhbHVlcyI6W251bGxdfSx7ImlkIjoiQ2F0ZWdvcnkiLCJ2YWx1ZXMiOlsiRXJyb3JQcm9uZSJdfSx7ImlkIjoiTGV2ZWwiLCJ2YWx1ZXMiOltudWxsXX0seyJpZCI6IlBhdHRlcm4iLCJ2YWx1ZXMiOlsxNzEwXX0seyJpZCI6IkF1dGhvciIsInZhbHVlcyI6W251bGxdfV0=)

**Comment by afuetterer** [comment]
Are there any updates on this issue?

**Comment by afuetterer** [comment]
6 more to go.

**Comment by afuetterer** [comment]
Can something go wrong in `.sub()`?
https://github.com/ddionrails/ddionrails/blob/8179e39fd4c87bd930b3d5840535f5e5e8fd4547/ddionrails/data/helpers.py#L128-L135

**Comment by afuetterer** [comment]
I think this can be deleted.
"description_long" is either a string or an empty string. `render_markdown()`.
Maybe the function `render_markdown` could throw some exception?
https://github.com/ddionrails/ddionrails/blob/8179e39fd4c87bd930b3d5840535f5e5e8fd4547/ddionrails/data/models/variable.py#L173-L178

**Comment by afuetterer** [comment]
If `self.description_long` is a string or an empty string, this should works just fine:
```python
 def html_description_long(self): 
     return render_markdown(self.description_long) 
```
description_long is a `TextField`, so it should not contain any other type instead of string.

**Comment by hansendx** [comment]
Some are now resolved with the merge of #1012
